iml-wasm-components/iml-stratagem/src/lib.rs

```diff
@@ -51,7 +53,7 @@ pub struct ActionResponse {
     command: iml_wire_types::Command,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
```

Comment:
This is now derived, but why do we need it at all?

Reply:
Will update this in a separate patch.